Source/Evolve/WarpXEvolve.cpp

```diff
@@ -375,50 +386,51 @@ WarpX::OneStep_nosub (Real cur_time)
     if (do_pml && pml_has_particles) CopyJPML();
     if (do_pml && do_pml_j_damping) DampJPML();
 
+    if (!do_electrostatic) {
+    // Electromagnetic solver:
     // Push E and B from {n} to {n+1}
     // (And update guard cells immediately afterwards)
 #ifdef WARPX_USE_PSATD
```

Comment:
The only changes below are the re-indentation fo the whole code block. 
Unfortunately, the Github interface does not show this very well.